### ISSUE TITLE
[Bug] add_jitter fails with AttributeError: other must be a LazyTensor at eval time

### ISSUE THREAD

**Issue by activatedgeek** [cornellius-gp/gpytorch#905]
# 🐛 Bug

Using `add_jitter` with the covariance module fails at eval time.

## To reproduce

** Code snippet to reproduce **

1. Start from the example https://github.com/cornellius-gp/gpytorch/blob/master/examples/05_Scalable_GP_Regression_Multidimensional/SVGP_Regression_CUDA.ipynb

2. Optionally, replace `WhitenedVariationalStrategy` with `VariationalStrategy` in `GPModel.__init__`

3. Replace `self.covar_module(x)` with `self.covar_module(x).add_jitter(1e-6)` in `GPModel.forward`

** Stack trace/error message **
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-6-c955f39ee560> in <module>
      4 with torch.no_grad():
      5     for x_batch, y_batch in test_loader:
----> 6         preds = model(x_batch)
      7         means = torch.cat([means, preds.mean.cpu()])
      8 means = means[1:]

~/miniconda3/envs/mbrl/lib/python3.7/site-packages/gpytorch/models/abstract_variational_gp.py in __call__(self, inputs, **kwargs)
     20             inputs = inputs.unsqueeze(-1)
     21 
---> 22         return self.variational_strategy(inputs)

~/miniconda3/envs/mbrl/lib/python3.7/site-packages/gpytorch/variational/variational_strategy.py in __call__(self, x)
    228                 self._memoize_cache = dict()
    229 
--> 230         return super(VariationalStrategy, self).__call__(x)

~/miniconda3/envs/mbrl/lib/python3.7/site-packages/gpytorch/module.py in __call__(self, *inputs, **kwargs)
     20 
     21     def __call__(self, *inputs, **kwargs):
---> 22         outputs = self.forward(*inputs, **kwargs)
     23         if isinstance(outputs, list):
     24             return [_validate_module_outputs(output) for output in outputs]

~/miniconda3/envs/mbrl/lib/python3.7/site-packages/gpytorch/variational/variational_strategy.py in forward(self, x)
    214                     induc_induc_covar.inv_matmul(induc_data_covar)
    215                 )
--> 216                 data_covariance = data_data_covar + neg_induc_data_data_covar
    217             predictive_covar = PsdSumLazyTensor(predictive_covar, data_covariance)
    218 

~/miniconda3/envs/mbrl/lib/python3.7/site-packages/gpytorch/lazy/sum_lazy_tensor.py in __add__(self, other)
     75             return SumLazyTensor(*(list(self.lazy_tensors) + [other]))
     76         else:
---> 77             raise AttributeError("other must be a LazyTensor")
     78 
     79     def diag(self):

AttributeError: other must be a LazyTensor
```

## Expected Behavior

The model to return outputs like always.

## System information

**Please complete the following information:**
- GPyTorch Version: `0.3.6`
- PyTorch Version: `1.2.0`
- OS Version: `Ubuntu 18.04.3 LTS`

**Comment by gpleiss** [comment]
Hi @activatedgeek - I believe this is going to be fixed as part of #903 . It might take a bit before that PR is in though.

Is there a reason that you want to add jitter to the covariance matrix as part of the forward function?

**Comment by activatedgeek** [comment]
Well, all my numerical instability issues have been resolved, so this isn't high priority for me. I just found this behavior and thought should report it anyways.
